docs/reference/java-interop.md

```diff
@@ -91,6 +107,12 @@ mnemonic notation for them:
 * `(Mutable)Collection<T>!` means "Java collection of `T` may be mutable or not, may be nullable or not",
 * `Array<(out) T>!` means "Java array of `T` (or a subtype of `T`), nullable or not"
 
+#### Nullability annotations
+
+Java types which have nullability annotations are represented not as platform types, but as actual nullable or non-null
+Kotlin types. Currently, the compiler supports the JetBrains flavor of the nullability annotations (`@Nullable` and `@NotNull`
+from the `org.jetbrains.annotations` package.
```

Comment:
Missing '(', also I guess would make sense to link to https://www.jetbrains.com/idea/help/nullable-and-notnull-annotations.html